plugins/woocommerce-blocks/assets/js/blocks/classic-template/archive-product.ts

```diff
@@ -66,9 +65,7 @@ const getBlockifiedTemplateWithTermDescription = (
 ) => getBlockifiedTemplate( inheritedAttributes, true );
 
 const isConversionPossible = () => {
-	// Blockification is possible for the WP version 6.1 and above,
-	// which are the versions the Products block supports.
-	return isWpVersion( '6.1', '>=' );
+	return true;
```

Comment:
With this change every `isConversionPossible` in templates return `true` now. What about removing the `isConversionPossible` completely and the related logic. I'm happy to contribute with that if you don't want to extend the scope of this PR.

Reply:
There are two instances with `false`

https://github.com/woocommerce/woocommerce/blob/4352324a699f6306f2274e483f9f1fb92f57b0c6/plugins/woocommerce-blocks/assets/js/blocks/classic-shortcode/index.tsx#L48

https://github.com/woocommerce/woocommerce/blob/4352324a699f6306f2274e483f9f1fb92f57b0c6/plugins/woocommerce-blocks/assets/js/blocks/classic-template/index.tsx#L53

I am afraid I don't have enough context here. I see it's used in two cases

https://github.com/woocommerce/woocommerce/blob/4352324a699f6306f2274e483f9f1fb92f57b0c6/plugins/woocommerce-blocks/assets/js/blocks/classic-shortcode/index.tsx#L189

https://github.com/woocommerce/woocommerce/blob/4352324a699f6306f2274e483f9f1fb92f57b0c6/plugins/woocommerce-blocks/assets/js/blocks/classic-template/index.tsx#L239

Reply:
I see what you mean! Most likely it can be removed but it would require more testing and verification so I created a separate issue which I'm happy to tackle during cooldown: https://github.com/woocommerce/woocommerce/issues/55957!

Reply:
Thanks for creating a follow up issue https://github.com/woocommerce/woocommerce/issues/55957